lesson6/README.md

```diff
@@ -0,0 +1,89 @@
+# Lesson 6 - How to Design an App
+
+##  Designing an app
+
+Here is an example process for desigining an app:
+
+#### 1. Observe
+Open up any app on your phone. It could be very simple or very complex; there's something to learn from both. While using it, ask questions. Some examples of questions could be:
```

Comment:
Should it only be an android app? IOS apps have slightly different design principles.  Or is this guide for general design, not just android design?